### PR TITLE
we have to move forward with grammar fixes

### PR DESCRIPTION
**Pull request by optimumtact**

### CODE REVIEW

**Comment by kevinz000** [comment]
its oranges

File: code/modules/uplink/uplink_item.dm [hunk 1]
```diff
@@ -297,7 +297,7 @@ GLOBAL_LIST_EMPTY(uplink_items) // Global list so we only initialize this once.
 /datum/uplink_item/dangerous/powerfist
 	name = "Power Fist"
 	desc = "The power-fist is a metal gauntlet with a built-in piston-ram powered by an external gas supply.\
-		 Upon hitting a target, the piston-ram will extend foward to make contact for some serious damage. \
+		 Upon hitting a target, the piston-ram will extend froward to make contact for some serious damage. \
```
Comment (YakumoChen) [inline code/modules/uplink/uplink_item.dm]: But.... That's not.... what....?

**Review by YakumoChen — CHANGES_REQUESTED** [review]

**Comment by ma44** [comment]
>tfw comment gets removed

**Comment by optimumtact** [comment]
mods are facists

File: code/modules/uplink/uplink_item.dm (hunk 1 above)
Comment (optimumtact) [inline code/modules/uplink/uplink_item.dm]: go with teh fro

**Pull request merged**